exercises/acronym/tests/acronym.rs

```diff
@@ -19,36 +19,57 @@ fn lowercase_words() {
 
 #[test]
 #[ignore]
-fn camelcase() {
-    assert_eq!(acronym::abbreviate("HyperText Markup Language"), "HTML");
+fn punctuation() {
+    assert_eq!(acronym::abbreviate("First In, First Out"), "FIFO");
 }
 
 #[test]
 #[ignore]
-fn punctuation() {
-    assert_eq!(acronym::abbreviate("First In, First Out"), "FIFO");
+fn all_caps_word() {
+    assert_eq!(acronym::abbreviate("GNU Image Manipulation Program"), "GIMP");
 }
 
 #[test]
 #[ignore]
-fn all_caps_words() {
-    assert_eq!(acronym::abbreviate("PHP: Hypertext Preprocessor"), "PHP");
+fn punctuation_without_whitespace() {
+    assert_eq!(
+        acronym::abbreviate("Complementary metal-oxide semiconductor"),
+        "CMOS"
+    );
 }
 
 #[test]
 #[ignore]
-fn non_acronym_all_caps_word() {
+fn very_long_abbreviation() {
     assert_eq!(
-        acronym::abbreviate("GNU Image Manipulation Program"),
-        "GIMP"
+        acronym::abbreviate("Rolling On The Floor Laughing So Hard That My Dogs Came Over And Licked Me"),
+        "ROTFLSHTMDCOALM"
     );
 }
 
 #[test]
 #[ignore]
-fn hyphenated() {
```

Comment:
please also restore this test case

Reply:
Not sure if you meant a different test case, but `hyphenate()` was just renamed to `punctuation_without_whitespace()`. They both check that "Complementary metal-oxide semiconductor" becomes "CMOS".

Reply:
Oh, oops! I missed that, sorry. Given a bunch of cases, I just grepped through the changes; I didn't think to grep for the actual acronym. I agree, consistency in name with the canonical data makes more sense.